assets/queries/googleDeploymentManager/cloud_storage_bucket_is_publicly_accessible/query.rego

```diff
@@ -0,0 +1,21 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	resource := input.document[i].resources[idx]
+	public_access_users := ["allUsers", "allAuthenticatedUsers"]
+	resource.type == "storage.v1.bucketAccessControl"
+	properties := resource.properties
```

Comment:
The variable `properties` defined in `properties := resource.properties` is not being used. It can be deleted.

Reply:
Done!